tests/test_locale_messages.py

```diff
@@ -0,0 +1,1017 @@
+import os
```

Comment:
This file is unreviewable. We'd have to update this all the time, and there isn't really a need to test for these messages.